### PR TITLE
transientStorage functions

### PR DESCRIPTION
**Pull request by Hrushi20**
Pull Request adds TransientStorage functions to fe based on issue #978

### CODE REVIEW

**Comment by g-r-a-n-t** [comment]
@Hrushi20 you can test locally using `cargo test` or `cargo test --features solc-backend`

I forgot to mention that solc-rust will need to be updated from `0.8.22` to `0.8.24`. this is usually a pretty straight forward task

File: crates/analyzer/src/builtins.rs [hunk 1]
```diff
@@ -73,6 +73,8 @@ pub enum Intrinsic {
     __sload,          // (p)
     __sstore,         // (p, v) -> ()
     __msize,          // ()
+    __tstore,         // ()
```
Comment (g-r-a-n-t) [inline crates/analyzer/src/builtins.rs:76]: the parameter and return count should be added to the functions defined in this module

**Comment by Hrushi20** [comment]
Sorry for the late fix. Got caught up with other works.

**Comment by Hrushi20** [comment]
Hey, how do I fix the broken tests?

**Pull request closed without merging**